exercises/xorcism/.meta/description.md

```diff
@@ -0,0 +1,69 @@
+Write a streaming adaptor which contains a reference to a key, and bitwise-xors it with arbitrary data.
+
+XOR is a fundamental binary operation: for each bit in the inputs, set the corresponding bit of the
+output to `1` if the input bits are different. If both inputs are `1` or both are `0`, then the corresponding
+output bit is `0`.
+
+XORing a document with a key has been used for cryptography as recently as the early 1900s. The key is repeated
```

Comment:
This was already obvious to me, but I will also attempt to think of this from the eyes of someone who might not have worked with crypto as much: "The key is repeated as many times as necessary" is a pretty important instruction because it specifies behaviour that is going to be tested. But the first sentence of this paragraph is background context and not important to the problem specification. Might this cause someone to assume the rest of the paragraph is of the same nature (background context) and thus miss the important instruction?

Reply:
Good point. Updated in 9aea97b.